OpenSim/Simulation/SimbodyEngine/SimbodyEngine.h

```diff
@@ -185,6 +162,85 @@ OpenSim_DECLARE_CONCRETE_OBJECT(SimbodyEngine, Object);
+    
+    /** <b>(Deprecated)</b> See Frame::getAccelerationInGround()[0]. */
+    DEPRECATED_14("see Frame::getAccelerationInGround()[0]")
+    void getAngularAccelerationBodyLocal(const SimTK::State& s, const PhysicalFrame &aBody, SimTK::Vec3& rAngAcc) const;
+    
+    /** <b>(Deprecated)</b> Use Frame::getTransformInGround() instead. */
+    DEPRECATED_14("use Frame::getTransformInGround() instead")
+    SimTK::Transform getTransform(const SimTK::State& s, const PhysicalFrame &aBody) const;
+
+    /** <b>(Deprecated)</b> Use Frame::expressVectorInAnotherframe() instead. */
+    DEPRECATED_14("use Frame::expressVectorInAnotherframe() instead") 
+    void transform(const SimTK::State& s, const PhysicalFrame &aBodyFrom, const double aVec[3], const PhysicalFrame &aBodyTo, double rVec[3]) const;
+    
+    /** <b>(Deprecated)</b> Use Frame::expressVectorInAnotherframe() instead. */
+    DEPRECATED_14("use Frame::expressVectorInAnotherframe() instead")
+    void transform(const SimTK::State& s, const PhysicalFrame &aBodyFrom, const SimTK::Vec3& aVec, const PhysicalFrame &aBodyTo, SimTK::Vec3& rVec) const;
+    
+    /** <b>(Deprecated)</b> Use Frame::findLocationInAnotherFrame() instead. */
+    DEPRECATED_14("use Frame::findLocationInAnotherFrame() instead")
+    void transformPosition(const SimTK::State& s, const PhysicalFrame &aBodyFrom, const double aPos[3], const PhysicalFrame &aBodyTo, double rPos[3]) const;
+    
+    /** <b>(Deprecated)</b> Use Frame::findLocationInAnotherFrame() instead. */
+    DEPRECATED_14("use Frame::findLocationInAnotherFrame() instead")
+    void transformPosition(const SimTK::State& s, const PhysicalFrame &aBodyFrom, const SimTK::Vec3& aPos, const PhysicalFrame &aBodyTo, SimTK::Vec3& rPos) const;
+    
+    /** <b>(Deprecated)</b> Use Frame::findLocationInGround() instead. */
+    DEPRECATED_14("use Frame::findLocationInGround() instead")
+    void transformPosition(const SimTK::State& s, const PhysicalFrame &aBodyFrom, const double aPos[3], double rPos[3]) const;
+    
+    /** <b>(Deprecated)</b> Use Frame::findLocationInGround() instead. */
+    DEPRECATED_14("use Frame::findLocationInGround() instead")
+    void transformPosition(const SimTK::State& s, const PhysicalFrame &aBodyFrom, const SimTK::Vec3& aPos, SimTK::Vec3& rPos) const;
+
+    /** <b>(Deprecated)</b> Use Point::calcDistanceBetween() or Frame::findLocationInGround() instead */
+    DEPRECATED_14("use Point::calcDistanceBetween() or Frame::findLocationInGround() instead")
+    double calcDistance(const SimTK::State& s, const PhysicalFrame& aBody1, const SimTK::Vec3& aPoint1, const PhysicalFrame& aBody2, const SimTK::Vec3& aPoint2) const;
+    
+    /** <b>(Deprecated)</b> Use Point::calcDistanceBetween() or Frame::findLocationInGround() instead */
+    DEPRECATED_14("use Point::calcDistanceBetween() or Frame::findLocationInGround() instead")
```

Comment:
The deprecated methods look great. I think we could also go ahead and deprecate the `SimbodyEngine` class as well so that users know to avoid using the class altogether.